iothub/device/src/Edge/TrustBundleProvider.cs

```diff
@@ -1,119 +1,24 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-using System;
-using System.Collections.Generic;
-using System.Diagnostics;
-using System.Linq;
-using System.Net.Http;
-using System.Net.Security;
```

Comment:
As per the C# SDK conventions (as in other files..), the usings are outside the namespaces. Let's keep the same conventions..

Reply:
done

---
In reply to: [193910901](https://github.com/Azure/azure-iot-sdk-csharp/pull/501#discussion_r193910901) [](ancestors = 193910901)